public/app/plugins/panel/geomap/layers/data/markersLayer.tsx

```diff
@@ -87,21 +91,28 @@ export const markersLayer: MapLayerRegistryItem<MarkersConfig> = {
           const dot = new Feature({
               geometry: info.points[i],
           });
-
+          // choose from dropdown of regular shapes
+          const shapesArr = shapes(fillColor)
+          if (config.icon) {
```

Comment:
given that we currently are not actually setting an icon... should this be `config.shape` and use `circle` as the default?

Reply:
Either radio or select:
```
.addRadio({
        path: 'mode',
        name: 'Mode',
        description: 'text mode of the panel',
        settings: {
          options: [
            { value: TextMode.Markdown, label: 'Markdown' },
            { value: TextMode.HTML, label: 'HTML' },
          ],
        },
        defaultValue: defaultPanelOptions.mode,
      })
      ```

Reply:
I think a select would make more sense here, vs a radio, and have circle as default in the select. I left the `icon` in case we want to change this later back to our list of icon, but can change to shape instead.